Samsung/LPVS
# Add login page

# Description

LPVS Front-End Login Page.
User, History, and Dashboard pages will be uploaded.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?
For using social login with Naver, Kakao, and Google, fill the necessary information:

- Update the `application.properties` located in `src/main/resources`:

Enter client-id and client-secret. (Kakao doesn't require a client-secret.)
Naver: https://developers.naver.com/products/login/api/api.md
Kakao: https://developers.kakao.com/docs/latest/ko/kakaologin/common
Google: https://cloud.google.com/identity-platform/docs/web/google

- Update the `.env` file located in `frontend`:

Enter your REST_API_KEY.

```
cd frontend
npm install
npm start
```
1. Use Chrome to navigate to http://localhost:3000
![home](https://github.com/Samsung/LPVS/assets/57388014/f2f128f4-0691-4780-9dae-366cf3ee731f)
2. On the Home Page, click Login in the upper right corner.
3. You can login to any of the three places you want, Naver, Kakao, and Google.
![Login](https://github.com/Samsung/LPVS/assets/57388014/9054bd7e-aacc-43c1-b298-e1a7427dd713)
4. After logged in, you can check your name in the upper right corner of the Home Page.
![home_login](https://github.com/Samsung/LPVS/assets/57388014/ac826c61-c5c9-4b4f-9691-123aeb4966f9)


**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:
